src/pages/office/office.ts

```diff
@@ -81,9 +86,11 @@ import {
 import { MultiFaultAssignableCompetency, CommentedCompetency } from '../../shared/models/fault-marking.model';
 import { OutcomeBehaviourMapProvider } from '../../providers/outcome-behaviour-map/outcome-behaviour-map';
 import { behaviourMap } from './office-behaviour-map';
+import { TerminationCode, terminationCodeList } from './components/termination-code/termination-code.constants';
 import { TestStatusCompleted } from '../../modules/tests/test-status/test-status.actions';
 
 interface OfficePageState {
+  terminationCode$: Observable<TerminationCode>;
```

Comment:
Perhaps activity code would be a better term here? We could already have pass/fail activity code, which isn't a termination code.